k8s/operators/src/diskpool/crd.rs

```diff
@@ -56,6 +56,8 @@ pub enum PoolState {
     /// The resource is present, and the pool has been created. The schema MUST
     /// have a status and spec field.
     Online,
+    /// This state is set when we receive delete event on the dsp cr.
+    Terminating,
```

Comment:
I thought we discussed yesterday not doing this and instead separating into 2 different states? Am I misremembering @blaisedias ?

Reply:
@tiagolobocastro, Now i have added one more field i.e status. This will represent control plane status for the pool.